_shared/project/tox.ini

```diff
@@ -84,18 +90,12 @@ allowlist_externals =
     {{- include("tox/allowlist_externals", indent=4) -}}
 {% endif %}
 {% endif %}
-{% if cookiecutter._directory in ['pyapp', 'pyramid-app'] or include_exists("tox/commands_pre") %}
-commands_pre =
-    {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
-    pip-sync requirements/{env:TOX_ENV_NAME}.txt --pip-args '--disable-pip-version-check'
-    {% endif %}
-    {% if include_exists("tox/commands_pre") %}
-        {{- include("tox/commands_pre", indent=4) -}}
-    {% endif %}
-{% endif %}
 commands =
+{% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
+    pip-sync-faster requirements/{env:TOX_ENV_NAME}.txt --pip-args '--disable-pip-version-check'
+{% endif %}
 {% if cookiecutter.get("postgres") == "yes" %}
-    tests: sh bin/create-db {{ cookiecutter.package_name }}_test
+    tests: sh bin/create-db {{ cookiecutter.package_name }}_tests
```

Comment:
I renamed the unit test DB from `lms_test` to `lms_tests` to avoid some `test` versus `tests` awkwardness that it would otherwise cause